app/controllers/movies_actors_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MoviesActorsController < ApplicationController
+
+
+  def create
+    movie = Movie.find(params[:id])
+    actor = Actor.find(params[:actor_id])
+    MoviesActor.create!(movie: movie, actor: actor)
```

Comment:
This is totally fine. You could also use ids/params directly, however if you prefer the readability that you have here, that is a completely valid design choice.